### DISCUSSION TITLE
Recorded Meetings for January 2023

### DISCUSSION THREAD

**Discussion by asher-lab** [okd-project/okd#1468] [General]
Hi, I noticed no new video recordings for 2023 are available in our youtube playlist. This is the link I am following: https://www.youtube.com/playlist?list=PLaR6Rq6Z4Iqc3WjZB-rUTPru8RKyOCnBo

**Comment by JaimeMagiera** [comment]
Thanks for reaching out. We had our first meeting of the year last Tuesday. That meeting, and today's meeting, will be posted in the next day or two.

**Comment by JaimeMagiera** [comment]
The recording for last week's meeting:

https://youtu.be/h0eKfvNgaHg

**Reply by asher-lab** [reply]
Thanks!

**Comment by JaimeMagiera** [comment]
And the Working Group meeting from January 17th:

https://youtu.be/CVJqTr5Yv4s
